CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.9.0-alpha+14
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+14. The alpha tag represents the evolving nature of the
+> AngularDart API, not code quality (5.0.0-alpha+14 is used in production Google
+> apps).
+
+ * Add Material Stepper component.
+ * Material Button: Update size and position of text labels for icon buttons.
+ * Material Checkbox: Migrate from the deprecated Glyph component to Material
+   Icon.
+ * Material Input: Don't override unfocused underline color when the input is
+   invalid.
+ * Material Select:
+   * Support focus on dropdown button when dismissed via keyboard.
+   * Fix focus bug when clicking on a different focusable element.
+ * Add new CSS class names used by `ShowHideDirective` to avoid conflicts with
+   Angular JS apps.
+ * Only focus when `_autoFocus` is `true` when using `AutoFocusDirective`.
```

Comment:
`_autoFocus` is private. Is there a public flag/option/something here?

Reply:
There is a public setter for autoFocus which is also an input. Probably just remove the _ in the release notes.